BudgetSquirrel.Api/ClientApp/src/app/budgets/add-budget-form/add-budget-form.component.html

```diff
@@ -0,0 +1,40 @@
+<div class="modal__background">
+  <dialog class="modal" open="true">
+
+    <div class="modal__header">
+      <h1 class="modal__title">New Budget Line Item</h1>
+      <button class="modal__close-btn" (click)="onCloseClicked()">
+        <svg class="close-btn-icon" xmlns="http://www.w3.org/2000/svg" height="28" viewBox="0 0 24 24" width="28">
```

Comment:
This can refactored later but I usually like to but svg into the `/assets` folder and then just do an `<image />` tag to load it in.

Reply:
Then we can't style the svg. I need to add the fill property on it but if it's an img, I can't do that.